examples/bq-email-exports/export_query_results_function/main.py

```diff
@@ -60,4 +66,10 @@ def main(event, context):
 
 def get_destination_uri():
     """Returns destination GCS URI for export"""
-    return f"gs://{os.environ.get('BUCKET_NAME')}/{os.environ.get('OBJECT_NAME')}"
+    return (f"gs://{get_env('BUCKET_NAME')}/"
+            f"{time.strftime('%Y%m%d-%H%M%S')}/{get_env('OBJECT_NAME')}")
```

Comment:
Nit: This is not a very common way to represent a timestamp https://docs.python.org/3/library/datetime.html#datetime.datetime.timestamp. I'd suggest just using a unix timestamp which is most explicit. or having something like `yyyy=%Y/mm=%m/dd=%d/hr=%H/` which will be familiar to Hadoop users.

Reply:
Also currently this is cloud function execution time. Would it make more sense to use the scheduled time of the query (read from the transfer run)?

Reply:
Yeah, changed it to use the schedule time from BTS in ISO time format